web-client/src/presenter/computeds/practitionerInformationHelper.test.ts

```diff
@@ -1,9 +1,64 @@
+import {
+  PRACTICE_TYPE,
+  ROLES,
+} from '@shared/business/entities/EntityConstants';
+import { PractitionerDetail } from '@web-client/presenter/state';
 import { applicationContext } from '../../applicationContext';
 import { practitionerInformationHelper as practitionerInformationHelperComputed } from './practitionerInformationHelper';
 import { runCompute } from '@web-client/presenter/test.cerebral';
+import { v4 } from 'uuid';
 import { withAppContextDecorator } from '../../withAppContext';
 
 describe('practitionerInformationHelper', () => {
+  const pageSize = 100; // This should match the page size in practitionerInformationHelper
```

Comment:
Another tiny nit: could this test and `practitionerInformationHelper` use the same exported `PAGE_SIZE` constant?

Reply:
Sure! I always waffle between exporting the constant (which keeps the test coupled to the code) and not (which means we are testing that the constant is the value we expect).

Reply:
Fair point, dealer's choice!

Reply:
I went with your suggestion, which is usually what I end up doing in these cases. Thank you!